docs/fields.asciidoc

```diff
@@ -558,6 +558,56 @@ type: alias
 --
 
 
+*`meta.cloud.availability_zone`*::
```

Comment:
Note that work is already underway to make the aliases render better: https://github.com/elastic/beats/pull/9269 and https://github.com/elastic/beats/issues/9288